hack/gen-tasks.go

```diff
@@ -142,15 +142,16 @@ func main() {
 	for _, test := range []string{
 		"",
 		"InstallationSuite",
-		"VerifySuite",
 		"Integration",
-		"Integration/derivedCR",
-		"Integration/serviceCluster",
-		"Integration/catalog",
 		"Integration/account",
+		"Integration/api-server",
+		"Integration/catalog",
 		"Integration/cli",
+		"Integration/derivedCR",
+		"Integration/serviceCluster",
 		"Scenarios",
 		"Scenarios/simple",
+		"VerifySuite",
```

Comment:
Why this `VerifySuite` is moved to the last test?

Reply:
alphabetic sorting, as it should have been done in the first place.